features/queues.feature

```diff
@@ -0,0 +1,99 @@
+Feature: Queue support
+
+@not-laravel-latest @not-laravel51 @not-laravel56 @not-laravel58 @not-laravel66 @not-laravel8 @not-lumen8
```

Comment:
It may be easier at this point to have a whitelist instead of a blacklist function for skipping tests